src/webots/vrml/WbProtoTreeItem.hpp

```diff
@@ -30,6 +30,9 @@ class WbProtoTreeItem : public QObject {
   const WbProtoTreeItem *parent() const { return mParent; }
   const QList<WbProtoTreeItem *> children() const { return mChildren; }
 
+  void setRawUrl(const QString &url) { mRawUrl = url; }
+  const QString &rawUrl() const { return mRawUrl; }
```

Comment:
It seem this method is never called, and the value of `mRawUrl` is also never used.